source/driver-models/Timer.cpp

```diff
@@ -225,6 +225,7 @@ int Timer::eventEveryUs(CODAL_TIMESTAMP period, uint16_t id, uint16_t value)
  */
 void Timer::sync(CODAL_TIMESTAMP t)
 {
+    if (t > 3600000000LL) for(;;);
```

Comment:
Is this required @mmoskal ? Wouldn't it "just wrap around"?

Reply:
This is left-over debugging. This code sometimes does indeed trigger, due to some races somewhere. We get a negative number, which results in a number larger than 63 bits (so much larger than an hour, stated here). This is particularly bad if you use the CPX-specific timer, which only disables timer IRQ not all IRQs. We don't use that timer in PXT though.

Anyway, will remove.